### PR TITLE
Create inheritance.md

### LINKED ISSUE #629: [Entry/Edit] CSS: Inheritance
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Add if the CSS property is inherited.

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added the CSS inheritance entry.

Closes #629 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
Dont merge until #640 is.

File: content/css/concepts/inheritance/inheritance.md
Comment (SSwiniarski) [inline content/css/concepts/inheritance/inheritance.md:3]: I think the phrase "when the parent is the selector" is making the sentence less clear. Maybe something more like: Styling that is applied to a chilled element when they are applied to its parent.

File: content/css/concepts/inheritance/inheritance.md
Comment (SSwiniarski) [inline content/css/concepts/inheritance/inheritance.md:18]: See my prior comment.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ethan-master-coding, I did the first review and left a couple of comments.

**Comment by ethan-master-coding** [comment]
@SSwiniarski All edits look good. I accepted the changes and made the requested changes.

File: content/css/concepts/inheritance/inheritance.md
Comment (Dusch4593) [inline content/css/concepts/inheritance/inheritance.md:33]:
```suggestion
| All [background](https://www.codecademy.com/resources/docs/css/background) properties | Adds wallpaper visuals for HTML elements. | 
```
I would remove the backticks since this is linking to the CSS background concept and not the actual `background` property term entry. This suggestion also applies to some more rows down below.

File: content/css/concepts/inheritance/inheritance.md
Comment (Dusch4593) [inline content/css/concepts/inheritance/inheritance.md:34]:
```suggestion
| All [border](https://www.codecademy.com/resources/docs/css/borders) properties | Defines the border styles of an element. |
```

File: content/css/concepts/inheritance/inheritance.md
Comment (Dusch4593) [inline content/css/concepts/inheritance/inheritance.md:47]:
```suggestion
| [`box-sizing`](https://www.codecademy.com/resources/docs/css/box-sizing) | Includes `padding` and `border` in the calculation of an element's `height` and `width`. |
| All [Flexbox](https://www.codecademy.com/resources/docs/css/flexbox) properties | Arranges elements responsively and according to screen size. |
| [`float`](https://www.codecademy.com/resources/docs/css/floats) | Repositions an element to the left or right side of its containing element. |
| All [Grid](https://www.codecademy.com/resources/docs/css/grids) properties | Uses rows, columns, and gaps to organize content on a web page. |
| All [`margin` properties](https://www.codecademy.com/resources/docs/css/margins) | Sets the spacing beyond the border of an element. |
| All [overflow properties](https://www.codecademy.com/resources/docs/css/overflow) |https://www.codecademy.com/resources/docs/css/background | Manages content that spills  outside of its box. |
| All [padding](https://www.codecademy.com/resources/docs/css/padding) properties | Sets the spacing between the content and margin of an element. |
| All [position](https://www.codecademy.com/resources/docs/css/position) properties | Defines the placement of an element on a web page. |  
| All [sizing](https://www.codecademy.com/resources/docs/css/sizing) properties | Defines the `height` and `width` values of an element. |
| [`text-decoration`](https://www.codecademy.com/resources/docs/css/typography/text-decoration) | Styles the text of an element with line decorations. |
| [`text-overflow`](https://www.codecademy.com/resources/docs/css/typography/text-overflow) | Specifies how hidden content is signaled to the user. |
| [`transform`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) |
| [`transform-origin`](https://www.codecademy.com/resources/docs/css/transform-functions/transform-origin) | Sets the position of a transformed element relative to its origin. |
| [`transform-style`](https://www.codecademy.com/resources/docs/css/transform-functions/transform-style) | Confirms whether an element's children are flattened or positioned in a 3D space. |
| All [transition](https://www.codecademy.com/resources/docs/css/transition) properties | Controls the animation speed and timing of property changes in an element. |
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding Went ahead and left some comments/suggestions for my review. I implicitly did this in the suggested edits for the _second_ markdown table where all the properties are now alphabetized. Let's do the same thing for the first markdown table beginning on line # 21.

**Review by SSwiniarski — APPROVED** [review]

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes and made the requested changes.

File: content/css/concepts/inheritance/inheritance.md
Comment (Dusch4593) [inline content/css/concepts/inheritance/inheritance.md:24]: Super nit-picky suggestion but could we make sure the rows in these tables are alphabetized by "Property", such as moving this row for `visibility` down below? (between `text-transform` and `word-spacing`)

**Review by Dusch4593 — APPROVED** [review]
@ethan-master-coding With exception of the one comment I have left for you, everything else LGTM 👍🏻

We should get one more reviewer to look at this before merging since we're working with two markdown tables and a lot of links.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Made it alphabetized.

**Comment by ethan-master-coding** [comment]
@KTom101 I fixed it

**Review by KTom101 — APPROVED** [review]
@ethan-master-coding Great! @Dusch4593 I think this is ready to merge!

**Pull request merged**
